system/services/admin/DataManagerService.cfc

```diff
@@ -766,6 +769,13 @@ component {
 		var poService            = _getPresideObjectService();
 		var relationshipGuidance = _getRelationshipGuidance();
 		var searchTerms          = arguments.expandTerms ? listToArray( arguments.q, " " ) : [ arguments.q ];
+		var objEnumFields        = poService.getObjectEnumProperties( objectName=arguments.objectName );
```

Comment:
This checks if an object has enum properties and gets them all - whether or not they are included in the fields to be searched.

See comment below for an alternative way to check the enum fields.